src/components/ScrollTop/ScrollTop.js

```diff
@@ -3,7 +3,10 @@ import styled from '@emotion/styled';
 import Arrow from 'images/up-arrow.inline.svg';
 import { onMobile, onTablet } from '../../styles/responsive';
 
-const scrollToTop = () => window.scrollTo(0, 0);
+const scrollToTop = () => {
+  const element = document.getElementById('scroll-to-top-main-content');
+  element?.scrollTo(0,0)
```

Comment:
.? nice

Reply:
on production version is bug, the `ScrollToTop` button is broken.
Fix `ScrollToTop` components to work properly. (I forget to write number 5),